### ISSUE TITLE
Duplicated abstract information in data

### ISSUE THREAD

**Issue by ronaldtse** [ietf-ribose/relaton-data-ieee#10]
https://github.com/ietf-ribose/relaton-data-ieee/blob/d013c6b67ac1db02c1c0a19e02ec2638a0f7400c/data/AIEE_11.1937.yaml#L47-L62

**Comment by andrew2net** [comment]
@ronaldtse the [source](https://github.com/ietf-ribose/ieee-rawbib/blob/5c65670459cac47f014225c4d1415aeb0eafb334/cache/IEEEstd/1937/7361732/7361733/07361734.xml#L79-L80) has 2 abstracts. Should we parse only one?

**Comment by ronaldtse** [comment]
@andrew2net yes we only need to parse one. Can you check if other items also have this problem and produce a list?
